app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -397,7 +397,7 @@ class Contact extends React.Component {
                 </Form.Group>
               </Form.Row>
             </Form>
-            {this.props.previous && (
+            {this.props.previous && !this.props.hidePreviousButton && !this.state.isEditMode && (
```

Comment:
same here

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9